### ISSUE TITLE
Python 3

### ISSUE THREAD

**Issue by RyanBalfanz** [tdeck/teambot#8]
I'm willing to do this work too.

**Comment by tdeck** [comment]
Good idea. I'm working to de-crapify rtmbot.py so there's potentially less code to convert. It's doing a lot of things that we don't actually need and I never cleaned it up because nobody ever worked on this code before :).

**Comment by tdeck** [comment]
It looks like the only change is s/iteritems/items/, according to 2to3. I'm testing it out!

**Comment by RyanBalfanz** [comment]
Sweet, don't forget to add Python 3.6, etc. to `.travis.yml`. :)

**Comment by tdeck** [comment]
I played with this a bit in the python3 branch (pushed). Noticed a couple of issues:
- After starting in Python 3, teambot couldn't read the DB file I'd previously created under Python 2.7. So that does make migration potentially tougher for some people. If anyone's actually using this and migrating an existing database to Python 3, I'm sure we can write a script or some kind of export functionality to make it easier.
- The slackclient lib we depend on [doesn't support Python 3](https://github.com/slackapi/python-slackclient#requirements-and-installation). I get an error when running it. It's a bit puzzling since it's 2017 and this is an official Slack library, but they do have [a bug open for it](https://github.com/slackapi/python-slackclient/issues/165).
